keyboards/jpe230/big_knob/big_knob.c

```diff
@@ -0,0 +1,167 @@
+// Copyright 2023 jpe230 (@jpe230)
```

Comment:
If all the custom init code is just for inversion, you could use 2/3 lines of code for it (`qp_comms{start, databyte, stop}`). I have an open PR for that if you wanna check it and/or provide feedback.

Reply:
Sure, will test it :) kinda whack I have to duplicate the init. code just to invert colors :p